unit/util/interval/add.cpp

```diff
@@ -0,0 +1,95 @@
+/*******************************************************************\
+ Module: Unit tests for variable/sensitivity/abstract_object::merge
+ Author: DiffBlue Limited
```

Comment:
So it turns out it’s spelled _Diffblue_ actually, oops.

Reply:
Will fix.